mac/keyrace-mac/keyrace_macApp.swift

```diff
@@ -102,13 +102,18 @@ class KeyTap {
         if (lastDay != day) {
             lastDay = day
             keycount = 0
-            minutes = [Int](repeating:0, count:1440)
+            minutes.replaceSubrange(0..<1420, with: repeatElement(0, count: 1420)) // Leave last 20 minutes of yesturday to show on chart
             keys = [Int](repeating:0, count:256)
         }
         
         keycount += 1
         
         let hour = calendar.component(.hour, from: date)
+        
+        if (0 < hour && hour < 23) {
+            minutes.replaceSubrange(1420..<1440, with: repeatElement(0, count: 20)) // Deletes the last 20 minutes of the previous day
```

Comment:
Would probably be better not to run this on every keypress between 1am and 11pm :-)